test/controllers/quote_controller_test.exs

```diff
@@ -44,7 +44,52 @@ defmodule HonGraffitiPhoenix.QuoteControllerTest do
 
     assert json_response(conn, 422) ==
       render_changeset_json("error.json", changeset: invalid_quote_changeset)
-    refute Repo.get_by(Quote, raw: params_for(:quote).raw)
+    refute Repo.get_by(Quote, raw: params_for(:invalid_quote).raw)
+  end
+
+  test "#update renders the updated quote and persists a valid quote" do
+    conn = build_conn()
+    original_quote_attrs = %{raw: "^gOriginal quote"}
+    updated_quote_attrs = %{raw: "^yUpdated quote"}
+    original_quote = insert(:quote, original_quote_attrs)
+
+    conn = patch(
+      conn,
+      quote_path(conn, :update, original_quote),
+      quote: params_for(:quote, updated_quote_attrs)
+    )
+
+    updated_quote = Repo.get_by(Quote, raw: updated_quote_attrs.raw)
+    assert json_response(conn, 200) ==
+      render_json("show.json", quote: updated_quote)
+    refute Repo.get_by(Quote, original_quote_attrs)
```

Comment:
No change required. Is refuting the old value better than asserting the new value?
This is just a perspective thing